src/lv_conf_kconfig.h

```diff
@@ -10,35 +10,36 @@
 extern "C" {
 #endif
 
+#if defined ESP_PLATFORM
+#include "sdkconfig.h"
+#include "esp_attr.h"
+#endif
+
+/*******************
+ * LV COLOR TRANSP
+ *******************/
+
+#ifndef LV_COLOR_TRANSP
+ #if defined (CONFIG_LV_COLOR_TRANSP_HEX)
+  #define CONFIG_LV_COLOR_TRANSP    lv_color_hex(CONFIG_LV_COLOR_TRANSP_HEX)
+ #else
+  #define CONFIG_LV_COLOR_TRANSP    LV_COLOR_LIME
+ #endif
+#endif
+
 /*******************
  * THEME SELECTION
  *******************/
  
 #ifndef LV_THEME_DEFAULT_INIT
  #if defined (CONFIG_LV_THEME_DEFAULT_INIT_EMPTY)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_empty_init
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_empty_init
  #elif defined (CONFIG_LV_THEME_DEFAULT_INIT_TEMPLATE)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_template_init
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_template_init
  #elif defined (CONFIG_LV_THEME_DEFAULT_INIT_MATERIAL)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_material_init
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_material_init
  #elif defined (CONFIG_LV_THEME_DEFAULT_INIT_MONO)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_mono_init
- #endif
-#endif
-
-/*******************
- * COLOR SELECTION
- *******************/
- 
-#ifdef LV_THEME_DEFAULT_COLOR_PRIMARY
- #ifdef CONFIG_LV_THEME_DEFAULT_COLOR_PRIMARY
-  #define LV_THEME_DEFAULT_COLOR_PRIMARY lv_color_hex(CONFIG_LV_THEME_DEFAULT_COLOR_PRIMARY) 
- #endif
-#endif
- 
-#ifdef LV_THEME_DEFAULT_COLOR_SECONDARY
- #ifdef CONFIG_LV_THEME_DEFAULT_COLOR_SECONDARY
-  #define LV_THEME_DEFAULT_COLOR_SECONDARY lv_color_hex(CONFIG_LV_THEME_DEFAULT_COLOR_SECONDARY) 
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_mono_init
```

Comment:
Here's the definition of `CONFIG_LV_THEME_DEFAULT_INIT` when the mono theme is selected.